samples/app-dev/API-Development/REST/allOf_oneOf-anyOf_construct_schema/readme.md

```diff
@@ -0,0 +1,125 @@
+
+## Import the sample
+1. Download the sample json files i.e., *AllOf_OneOf_AnyOf_Keywords_APISpec_Service.json* and *Invoke_AllOf_OneOf_AnyOf_Keywords_APISpec_Service.json* 
+Rest triggers in the service app have been configured with Open API Specs 3.0 files. These API Specs are available in the *OpenAPISpecs* folder.
+
+2. Create a new empty app.
+![Create an app](../../import-screenshots/2.png)
+
+3. On the app details page, select Import app.
+![Select import](../../import-screenshots/3.png)
+
+4. Browse on your machine or drag and drop the .json file for the app that you want to import.
+![Import your sample](../../import-screenshots/4.png)
+
+5. Click Upload. The Import app dialog displays some generic errors and warnings as well as any specific errors or warnings pertaining to the app you are importing. It validates whether all the activities and triggers used in the app are available in the Extensions tab.
+![The Import app dialog](../../import-screenshots/5.png)
+
+6. You have the option to import all flows from the source app or selectively import flows.
+
+7. Click Next. If you had not selected a trigger in the previous dialog, the flows associated with that trigger are displayed. You have the option to select one or more of these flows such that the flows get imported as blank flows that are not attached to any trigger. By default, all flows are selected. Clear the check box for the flows that you do not want to import. If your flow(s) have subflows, and you select only the main flow but do not select the subflow, the main flow gets imported without the subflow. Click Next.
+
+
+## Understanding the configuration
+After importing the 'AllOf_OneOf_AnyOf_Keywords_APISpec_Service', we can see that multiple response codes have been configured in the Rest trigger. Few Response codes have Response headers as well. While adding schema in Response header and Response body we need to make sure that both are in JSON data or JSON Schema format.
+![MultipleResponseCode_configuration](./import-screenshots/MultipleResponse_serviceApp.png)
+
+ConfigureHTTPResponse activity should be used when we have configured multiple response codes in the Rest trigger. This activity is useful in mapping Response body and Response headers of a particular Response code in getting input from other activities and output to 'Return' activity.
+![ConfigureHTTPResponse_Input](./import-screenshots/ConfigureHTTPResponse_Input_Mappings.png)
+
+For the 'Invoke_AllOf_OneOf_AnyOf_Keywords_APISpec_Service' app, we have configured the activity with the API Spec of the 'AllOf_OneOf_AnyOf_Keywords_APISpec_Service'. We can upload swagger 2.0 or Open API spec 3.0 in the InvokeRestService activity to configure it. Request parameters, Request schema, Response schema and header would be auto-populated when a valid API spec file is uploaded. After uploading the spec we just need to map the input.
```

Comment:
Pls remove We can upload swagger 2.0, just mention OAS 3.0 service
Remove request parameters, Request schema, Response schema and header would be auto-populated when a valid API spec file is uploaded.